### ISSUE TITLE
NC license is not Open Source

### ISSUE THREAD

**Issue by hoijui** [rtitmuss/torn#26]
First of all, I am 100% sure you have noble reasons for it.
Secondly, I love your project; thank you! :-)

I know many people who use NC, because they are anti-capitalism.
Sadly, it is not that easy ...
What do you want to prevent by using NC?

* A chinese company mass producing your keyboard, and thus making a profit from your free work?
* Some one-man company offering a build service?
* mass-drops?
* ...?

All the really bad capitalists schemes I can imagine, would just use an other design, be it self-made or not.
Also, they would want an SMD design, as it has only benefits for them.
Practically, you prevent small/alternative businesses and individuals to supply a viable service to customers, which is not interesting for big companies.
in short: it is hurting all the wrong people.

someone elses take on that:
https://mifactori.de/non-commercial-is-not-open-source/#details-0-0

**Comment by keyboard-magpie** [comment]
This guy searches for NC projects and posts this all across them. It's a bit boring at this point.

**Comment by hoijui** [comment]
ahh... I may have done it once or twice so far, within the last 4 years or so.
also, I do not see what difference it makes.
